lblod/frontend-mow-registry
# GN-4353: disable search field trimming

## Overview
This PR removes the `.trim()` call when updating the search filter based on the value of an input field. It caused issues typing spaces into the input fields.

##### connected issues and PRs:
Jira ticket: https://binnenland.atlassian.net/browse/GN-4353?atlOrigin=eyJpIjoiNmQxNjkzMGQ5OGM0NGNmYzgxOGM5YmNkNWIyY2JlYWQiLCJwIjoiaiJ9


### How to test/reproduce
Visit any of the main index pages and type text and spaces into the search-fields, the inputs should no longer be trimmed.



### Checks PR readiness
